src/test/java/seedu/address/logic/commands/AddStudentToClassCommandTest.java

```diff
@@ -0,0 +1,136 @@
+import seedu.address.logic.commands.addstudenttoclasscommands.AddStudentToClassByIdCommand;
+import seedu.address.logic.commands.addstudenttoclasscommands.AddStudentToClassByIndexCommand;
+import seedu.address.logic.messages.PersonMessages;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.module.ModuleCode;
+import seedu.address.model.module.TutorialClass;
+import seedu.address.model.person.Email;
+import seedu.address.model.person.StudentId;
+
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for
+ * {@code DeleteCommand}.
+ */
+public class AddStudentToClassCommandTest {
+
+    private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+
+    @BeforeEach
+    public void setUp() {
+        ModuleCode newModule = new ModuleCode(VALID_MODULE_AMY);
+        model.addModule(newModule);
+        TutorialClass newTutorialClass = new TutorialClass(VALID_TUTORIAL_AMY);
+        newModule.addTutorialClass(newTutorialClass);
+    }
+
+    @Test
+    public void execute_invalidStudent_fail() {
+        AddStudentToClassByEmailCommand addStudentToClassByEmailCommand = new AddStudentToClassByEmailCommand(
+                new Email(INVALID_EMAIL), new ModuleCode(VALID_MODULE_AMY), new TutorialClass(VALID_TUTORIAL_AMY));
+        AddStudentToClassByIdCommand addStudentToClassByIdCommand = new AddStudentToClassByIdCommand(
+                new StudentId(INVALID_STUDENT_ID), new ModuleCode(VALID_MODULE_AMY),
+                new TutorialClass(VALID_TUTORIAL_AMY));
+
+        assertCommandFailure(addStudentToClassByEmailCommand, model,
+                String.format(PersonMessages.MESSAGE_PERSON_EMAIL_NOT_FOUND, INVALID_EMAIL));
+        assertCommandFailure(addStudentToClassByIdCommand, model,
+                String.format(PersonMessages.MESSAGE_PERSON_EMAIL_NOT_FOUND, INVALID_STUDENT_ID));
+    }
+
+
+    /* @Test
+    public void execute_duplicateStudent_fail() {
+        Person person = new PersonBuilder().build();
+        model.addPerson(person);
+        tutorialClass.addStudent(person);
+        AddStudentToClassByEmailCommand addStudentToClassByEmailCommand = new AddStudentToClassByEmailCommand(
+                person.getEmail(), new ModuleCode(VALID_MODULE_AMY), new TutorialClass(VALID_TUTORIAL_AMY));
+        AddStudentToClassByIdCommand addStudentToClassByIdCommand = new AddStudentToClassByIdCommand(
+                person.getStudentId(), new ModuleCode(VALID_MODULE_AMY),
+                new TutorialClass(VALID_TUTORIAL_AMY));
+        assertCommandFailure(addStudentToClassByIdCommand, model,
+                String.format(PersonMessages.MESSAGE_DUPLICATE_STUDENT_IN_CLASS, person, tutorialClass));
+        assertCommandFailure(addStudentToClassByEmailCommand, model,
+                String.format(PersonMessages.MESSAGE_DUPLICATE_STUDENT_IN_CLASS, person, tutorialClass));
+    } */
```

Comment:
Is this code block important? If not, it's probably best to remove it to adhere to coding standard/style